src/System.Private.ServiceModel/tools/CertificateGenerator/Program.cs

```diff
@@ -53,10 +53,24 @@ static void UninstallAllCerts()
             RemoveCertificatesFromStore(StoreName.Root, StoreLocation.CurrentUser);
         }
 
-        static void Main(string[] args)
+        static int Main(string[] args)
         {
             ApplyAppSettings();
 
+            if (args.Length > 0)
+            {
+                if (string.Compare(args[0], "-Uninstall", true) == 0)
+                {
+                    UninstallAllCerts();
+                    return 0;
+                }
+                else
+                {
+                    Console.WriteLine(string.Format("Argument {0} is not supported, the only supported argument is -Uninstall", args[0]));
```

Comment:
Nit: I would prefer having a Usage() method that shows valid usage of this command.  Then any problem can call it and exit.  Also consider supporting -? or -help to call Usage().